documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,373 @@
+
+[id="ova-provider-26_{context}"]
+=== OVA Provider
+
+This is a collection of new features for the {project-short} OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+.Get firmware detection by virt-v2v
+
+A `virt-v2v` that converts a guest also produces an OFV with configuration based on the virtual disk, such as driver or firmware. `forklift-controller` processes the produced OFV's status of the conversion pod. link:https://issues.redhat.com/browse/MTV-759[(MTV-759)]
+
+.Delete PV when the provider is deleted
+
+Each OVA provider has an attached PV and PVC. The PVC deletes automatically when the provider is deleted. {project-short} now also deletes the PV when the provider is deleted. link:https://issues.redhat.com/browse/MTV-848[(MTV-848)]
+
+.End-to-end test for import OVA
+
+The {project-short} process performs an end to end test of the imported OVA. link:https://issues.redhat.com/browse/MTV-624[(MTV-624)]
+
+.Create OVA provider under all projects
+
+The OVA provider can be created under all projects, not only under the forklift or MTV project. link:https://issues.redhat.com/browse/MTV-681[(MTV-681,] link:https://issues.redhat.com/browse/MTV-715[MTV-715)]
+
+.Scan files directly in share directory and down two sub-levels
+
+{project-short} scans files directly in the share directory and down two sub-levels. link:https://issues.redhat.com/browse/MTV-680[(MTV-680)]
+
+.Migrate from an OVA provider to a restricted project
+
+Migration from OVA provider to a restricted project succeeds past the `kubevirt` step. link:https://issues.redhat.com/browse/MTV-689[(MTV-689)]
+
+.OVA inventory watcher detects deleted files
+
+The OVA inventory watcher detects changes in files, including those that have been deleted. Updates from the `ova-provider-server` pod are sent every five minutes to the `forklift-controller` pod that updates the inventory. link:https://issues.redhat.com/browse/MTV-733[(MTV-733)]
+
+.Migration from OVA validation rules
+
+VM migrations from OVA are validated according to vSphere. The migration can use a similar structure and inventory variable names as vSphere, of which the OVA model is a subset. The rules do not apply to clusters, hosts, the running state of the VMs, or any configuration that is not relevant for the OVA VMs. link:https://issues.redhat.com/browse/MTV-669[(MTV-669)]
+
+.VM cannot find the bootable disk when source VMware provider uses VDDK 8
+
+When migrating from a source provider that uses VDDK (VMware Virtual Disk Development Kit) version 8, the migration completes successfully, but the converted guest cannot boot. The guest cannot find the boot disk. 
+
+For a complete list of all resolved issues in this release, see the list of link:https://issues.redhat.com/issues/?filter=12430274[Resolved Issues] in Jira.
+
+[id="known-issues-26_{context}"]
+== Known issues
```

Comment:
there's another known issue to add about migration of VMs with images from OpenStack to block storage, see last comments in https://issues.redhat.com/browse/MTV-644

Reply:
Add lines 270-272